### ISSUE TITLE
PersonAccount issues with listMetadata and deployment

### ISSUE THREAD

**Issue by SCWells72** [forcedotcom/cli#734]
I originally sent this to @ntotten, @clairebianchi, and @amphro directly and was asked to log it as an issue here:

I had the following issue filed against IC:

https://bitbucket.org/RoseSilverSoftware/illuminatedcloud/issues/1734

If you scan down you'll see that the issue is not actually with IC but rather with how certain `PersonAccount` metadata is returned by `listMetadata`. I've found the following known discrepancies, though it's very possible(/likely?) that there are others:

* **DuplicateRule**
    * `Account.Standard_Person_Account_Duplicate_Rule`
        * Full name should be `PersonAccount.Standard_Person_Account_Duplicate_Rule`
        * Filename should be `duplicateRules/PersonAccount.Standard_Person_Account_Duplicate_Rule.duplicateRule`
* **MatchingRule**
    * `Account.Standard_PersonAccount_Match_Rule_v1_0`
        * Full name should be `PersonAccount.Standard_PersonAccount_Match_Rule_v1_0`
        * Filename should be `matchingRules/PersonAccount.matchingRule`
* **RecordType**
    * `Account.PersonAccount`
        * Full name should be `PersonAccount.PersonAccount`
        * Filename should be `objects/PersonAccount.object`

Also note that deployment of duplicate rules seems to be broken in the CLI. When I try to deploy any duplicate rule--not just this weird `PersonAccount` one but any duplicate rule--`force:source:deploy` completes as follows:

```
$ sfdx force:source:deploy -m DuplicateRule:Account.Standard_Account_Duplicate_Rule --json
{
  "status": 0,
  "result": {
    "deployedSource": []
  }
}
```

And trying to deploy the actual file fails with:

```
$ sfdx force:source:deploy -p force-app/main/default/duplicateRules/Account.Standard_Account_Duplicate_Rule.duplicateRule-meta.xml
ERROR running force:source:deploy:  The DuplicateRule named Account.Standard_Account_Duplicate_Rule was not found in the workspace.
```

So obviously there are two issues included here, the first with the information reported by `listMetadata` and the second with deployment of duplicate rule metadata. Feel free to clone/split as appropriate.

**Comment by shetzel** [comment]
Hi @SCWells72 - the duplicate rules issue is known (#685 ) and the workaround is listed in that issue (add '# .forceignore v2' to your forceignore file).  That fix will be in either 50.6.0 or 50.7.0.
The other issue you mentioned with the listMetadata command is owned by another internal team so they would have to comment.

**Comment by mshanemc** [comment]
@SCWells72 I'm closing because I think #685 took care of this.  If the other listMetadata part is still bad, let us know.
